static/src/stylesheets/module/_social.scss

```diff
@@ -64,10 +67,71 @@ category: Common
     }
 }
 
+.social-icon.social-icon--more {
```

Comment:
Do you need the extra specificity here or can you just use `.social-icon--more`?